pyproject.toml

```diff
@@ -160,6 +160,28 @@ filterwarnings = [
     "ignore:SelectableGroups dict interface is deprecated:DeprecationWarning"
 ]
 
+[tool.setuptools]
+zip-safe = false
```

Comment:
https://setuptools.pypa.io/en/latest/deprecated/zip_safe.html

This line can just be removed